stream_alert/rule_processor/firehose.py

```diff
@@ -212,8 +212,29 @@ def firehose_request_wrapper():
                         record_batch_size,
                         stream_name)
 
-    @staticmethod
-    def _load_enabled_log_sources(firehose_config, log_sources):
+    def firehose_log_name(self, log_name):
```

Comment:
I should have caught this before, but this could still be a `classmethod` instead of converting from a staticmethod to an instance method (since the `SPECIAL_CHAR_REGEX` is a class constant). Not a crucial change